config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -672,6 +672,98 @@ presubmits:
         securityContext:
           privileged: true
 
+  # Fork of kubernetes/kubernetes: pull-kubernetes-e2e-gce-scale-performance-manual
+  - name: pull-perf-tests-clusterloader2-e2e-gce-scale-performance-manual
+    always_run: false
+    max_concurrency: 1
+    branches:
+    - master
+    decorate: true
+    path_alias: k8s.io/perf-tests
+    decoration_config:
+      timeout: 450m
+    extra_refs:
+    - org: kubernetes
+      repo: release
+      base_ref: master
+      path_alias: k8s.io/release
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-e2e-scalability-common: "true"
+      preset-e2e-scalability-presubmits: "true"
+    annotations:
+      testgrid-dashboards: presubmits-kubernetes-scalability
+      testgrid-tab-name: pull-perf-tests-clusterloader2-e2e-gce-scale-performance
+    run_if_changed: ^clusterloader2/.*$
```

Comment:
This line triggers this test in all PRs touching clusterloader2 (even if always_run: false).

https://github.com/kubernetes/test-infra/blob/0c4a2d5f9d2473e466d7c0e70f19a79bcfdca0f0/prow/config/jobs.go#L425

We can remove this single line.